test/test_functional_tensor.py

```diff
@@ -67,7 +67,6 @@ def test_scale_channel():
 
 
 class TestRotate:
-
```

Comment:
Is that a new `black` thing??

Reply:
It indeed comes from a newer `black` version that I have on my system. Since the removed blank lines don't conflict with the "old" style, it was not automatically reverted by running the `pre-commit` hook. I opened #7680 to upgrade our linters.